config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -0,0 +1,129 @@
+# This file contains pin mappings for the stock 2021 Creality Ender 3
+# S1. To use this config, during "make menuconfig" select the
+# STM32F103 with a "28KiB bootloader" and serial (on USART1 PA10/PA9)
+# communication.
+
+# If you prefer a direct serial connection, in "make menuconfig"
+# select "Enable extra low-level configuration options" and select
+# serial (on USART3 PB11/PB10), which is broken out on the 10 pin IDC
+# cable used for the LCD module as follows:
+# 3: Tx, 4: Rx, 9: GND, 10: VCC
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
```

Comment:
Where applicable this should be updated to reflect 
* The boards' versions, e.g. Creality 4.7.3.6.9.6 (They make a holy mess out of their board versions)
* The differences between the STM Chips (STM32F103RCT6 / STM32F401RCT6), e.g. bootloader offset, pins for USB / USART etc.
* Flashing procedure: Is folder `STM32F4_UPDATE` required for the F4 chip? Needs validation

Reply:
I forgot about that, will update.

Reply:
- CR-FDM-v24S1_301 (Next to the Creality logo on both the S1 and S1 Pro motherboard).
- However, in the settings menu of the S1 Pro, it states that H/W Ver is CR-FDM-v2.5.S1 

- F103 uses 28KiB bootloader offset
- F401 uses 64KiB bootloader offset
- Both F103 and F401 uses 8MHz crystal and USART1 PA10/PA9
- No need for STM32F4_UPDATE folder. Put bin file in root.

Reply:
Fixed now, I dont think the STM32F4_UPDATE folder is necessary, it seems others were able to flash just fine without it.